starlette/templating.py

```diff
@@ -38,6 +40,13 @@ def __init__(
                     "context": self.context,
                 }
             )
+
+        context = self.context.copy()
+        context.setdefault("request", request)
+
+        # TODO async?!
+        self.body = self.template.render(context).encode()
```

Comment:
I believe it should be async as template rendering involves IO (read template from disk).

Reply:
Yeah.. but that requires to use a different method / explicit setup with Jinja2 - but I agree that it should default to being async.

Reply:
This is not in scope for this PR though I'd say, just something I've noticed / noted there.

Reply:
https://jinja.palletsprojects.com/en/3.0.x/api/#jinja2.Template.render_async

It's available now.